perception/image_projection_based_fusion/config/pointpainting.param.yaml

```diff
@@ -1,10 +1,17 @@
 /**:
   ros__parameters:
-    class_names: ["CAR", "PEDESTRIAN", "BICYCLE"]
-    point_feature_size: 7  # x, y, z, time-lag and car, pedestrian, bicycle
+    class_names: ["CAR", "TRUCK", "BUS", "BICYCLE", "PEDESTRIAN"]
+    point_feature_size: 9  # x, y, z, time-lag and car, pedestrian, bicycle
     max_voxel_size: 40000
-    point_cloud_range: [-76.8, -76.8, -3.0, 76.8, 76.8, 5.0]
-    voxel_size: [0.32, 0.32, 8.0]
+    point_cloud_range: [-102.4, -102.4, -4.0, 102.4, 102.4, 6.0]
+    voxel_size: [0.32, 0.32, 10.0]
     downsample_factor: 1
-    encoder_in_feature_size: 12
+    encoder_in_feature_size: 14
     yaw_norm_thresholds: [0.3, 0.0, 0.3]
```

Comment:
Cloud you check the centerpoint config?
the size of `yaw_norm_thresholds` expectes to be the same as that of `class_names`

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/4314/commits/6b1e5df406e1e3028e00ea49116a2a18c4dffb9f#diff-43404fba97fdaccf59c9a9c3c8fc9f59a32831506a0342430b4457adb1cb243dR10